decidim-core/lib/decidim/core/test/shared_examples/map_examples.rb

```diff
@@ -58,19 +58,18 @@ def snippets
 shared_context "with frontend map elements" do
   let(:html_head) { "" }
   let(:html_document) do
-    builder = subject
     document_inner = html_body
     head_extra = html_head
+    builder_snippets = subject.append_assets
     template.instance_eval do
       <<~HTML.strip
         <!doctype html>
         <html lang="en">
         <head>
           <title>Map Test</title>
+          #{builder_snippets}
```

Comment:
```suggestion
```

You can remove this line because it is not adding anything to the template. I believe it is only an empty string.